microsoft/testsuites/power/stress.py

```diff
@@ -51,8 +52,8 @@ def before_case(self, log: Logger, **kwargs: Any) -> None:
     def stress_hibernation(self, environment: Environment, log: Logger) -> None:
         node = cast(RemoteNode, environment.nodes[0])
         is_distro_supported(node)
-        for _ in range(0, self._loop):
-            verify_hibernation(node, log)
+        for index in range(1, self._loop):
```

Comment:
why change this part? The `1` is a magic number, add comments above to explain it.